modules/ROOT/pages/bat-example-test-suite.adoc

```diff
@@ -0,0 +1,146 @@
+= Example of a Test Suite
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To see the content of an example test suite, in a terminal create a directory in your computer's filesystem. Then, change to that directory and run the `bat init` command. The command creates these folders and files:
+
+----
+.
+├── bat.yaml
+├── config
+│   ├── default.dwl
+│   ├── devx.dwl
+│   ├── qax.dwl
+│   └── stgx.dwl
+└── tests
+    └── HelloWorld.dwl
+----
+
+The `bat.yaml` file is the main file for the test suite. This example contains these lines:
+
+----
+suite:
+  name: "Hello World Suite"
+files:
+  - file: tests/HelloWorld.dwl
+reporters:
+    - type: HTML
+      outFile: /tmp/HTML.html
+  # - type: SumoLogic
+  #   options:
+  #     SUMO_ENDPOINT: <sumo_source>
+  # - type: PagerDuty
+  #   options:
+  #     PAGER_DUTY_ROUTING_KEY: <routing_key>
+  # - type: Slack
+  #   options:
+  #     SLACK_WEBHOOK: <webhook_url>
+  # - type: Email
+  #   options:
+  #     EMAILS: <email_list>
+  # - type: NewRelic
+  #   options:
+  #     NEW_RELIC_LICENSE_KEY: <license_key>
+  # - type: JSON
+  #   outFile: /tmp/JSON.json
+  # - type: JUnit
+  #   outFile: /tmp/JUnit.xml
+----
+
+Every test suite must contain a `bat.yaml` file at its root level.
+
+=== The `suite` Section of the example `bat.yaml` File
+The `suite` section names the suite. You can use any names that you like for the suites that you write.
+
+=== The `files` Section of the example `bat.yaml` File
+
+This example test suite contains only one test. The test is listed in the `files` section as the file `HelloWorld.dwl` in the `tests` folder.
+
+As you create tests, you add references to them in your `bat.yaml` file. For example, suppose that in your test-suite directory you create a folder named `api-tests` and write a test named `first-test.dwl` in that folder. In the `files` section of your `bat.yaml` file, you add the following indented line:
+
+----
+  file: api-tests/first-test.dwl
+----
+
+If you write a second test, this one named `second-test.dwl`, you add this line after that one:
```

Comment:
Can we avoid "this line after that one" and still retain intent?
```suggestion
If you write a second test, this one named `second-test.dwl`, you add this line next:
```

Reply:
Done.